third_party/nvidia/hopper/lib/Transforms/WarpSpecialization/CodePartitionUtility.cpp

```diff
@@ -428,38 +428,28 @@ unsigned getReuseAccumArgIdx(Operation *regionOp,
 // Compute and return the buffer index and phase for a given accumulate count.
 std::pair<Value, Value> getBufferIdxAndPhase(OpBuilderWithAsyncTaskIds &builder,
                                              Location loc, Value accumCnt,
-                                             unsigned numBuffers,
-                                             Operation *op) {
+                                             unsigned numBuffers) {
   Value numBuffersVal =
       builder.createWithAsyncTaskIds<arith::ConstantIntOp>(loc, numBuffers, 32);
-  copyLoopScheduleInfo(numBuffersVal.getDefiningOp(), op);
   numBuffersVal = builder.createWithAsyncTaskIds<arith::ExtSIOp>(
       loc, builder.getI64Type(), numBuffersVal);
-  copyLoopScheduleInfo(numBuffersVal.getDefiningOp(), op);
   // Calculate accumCnt / numBuffers
   // initBufferIdx = accumCnt - accumCnt / numBuffers * numBuffers
   // initPhase = (accumCnt / numBuffers) & 1
   Value bufferIdx = builder.createWithAsyncTaskIds<arith::DivUIOp>(
       loc, accumCnt, numBuffersVal);
-  copyLoopScheduleInfo(bufferIdx.getDefiningOp(), op);
```

Comment:
Unfortunately I didn't delete this function because the token lowering uses the old builder (I assume due to minimizing code changes). I didn't want to modify that too heavily.